src/components/common/AppLabel.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div class="label">
+    <label >
+      <slot></slot>
+    </label>
+  </div>
```

Comment:
Is there a reason why it's not simply just the `label` element with appropriate class? 🤔 
```suggestion
  <label class="label">
    <slot></slot>
  </label>
```

Reply:
Sure, no reason to pack it in the div here!